charts/synthetics-job-manager/charts/node-api-runtime/values.yaml

```diff
@@ -31,7 +31,13 @@ image:
 ## The AppArmor profile name that will be applied to the pods. If set, then the AppArmor profile must exist on the Kubernetes node(s) for this to work.
 # appArmorProfileName: ""
 
-resources: {}
+resources:
+  requests:
+    cpu: "500m"
+    memory: "1250Mi"
+  limits:
+    cpu: "750m"
+    memory: "2500Mi"
```

Comment:
Could you also put this same block up in the `synthetics-job-manager/values.yaml` file under the `node-api-runtime` section?

This will ensure that users who deploy everything via the root level synthetics-job-manager will be able to get the proper defaults. The values.yaml file here would most likely only be used if someone wanted to deploy this node-api-runtime subchart as a standalone deployment. So it still has value but I think we want it in both places.

Reply:
Nice catch! I feel so stupid for not seeing that, but it should be fixed now! Thank you!